src/e2e/Login.e2e.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const puppeteer = require('puppeteer');
+
+let browser;
+beforeAll(async () => {
+  browser = await puppeteer.launch({ headless: true });
+});
+
+describe('Login', () => {
+  test('Login fail', async () => {
```

Comment:
`failed`

Reply:
it would be better to add  a failed test case: login with empty username and password, because it has different error message and page presentation :)